modules/howtos/pages/subdocument-operations.adoc

```diff
@@ -217,7 +217,7 @@ bucket.MutateIn("my_array", ops =>
 [source,csharp]
 ----
 bucket.MutateIn("my_array", ops =>
-    ops.ArrayAppend('', ['elem1', 'elem2', 'elem3']”)
+    ops.ArrayAppend('', ['elem1', 'elem2', 'elem3']")
```

Comment:
Not sure if the `"` here is even necessary, but this is a `csharp` example even, so I expect this will change in future anyways 😄 .

Reply:
haha, good spot, oops. But yeah, it's not, erm, *any more broken* than it was before...